client/my-sites/email/email-management/home/email-plan.jsx

```diff
@@ -0,0 +1,319 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classnames from 'classnames';
+import { CompactCard } from '@automattic/components';
+import { connect } from 'react-redux';
+import { isEnabled } from '@automattic/calypso-config';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import wp from 'calypso/lib/wp';
+import {
+	getGoogleAdminUrl,
+	getGoogleMailServiceFamily,
+	getGSuiteProductSlug,
+	getGSuiteSubscriptionId,
+	getProductType,
+	hasGSuiteWithUs,
+} from 'calypso/lib/gsuite';
+import { getTitanSubscriptionId, hasTitanMailWithUs } from 'calypso/lib/titan';
+import HeaderCake from 'calypso/components/header-cake';
+import VerticalNav from 'calypso/components/vertical-nav';
+import VerticalNavItem from 'calypso/components/vertical-nav/item';
+import EmailTypeIcon from 'calypso/my-sites/email/email-management/home/email-type-icon';
+import {
+	emailManagement,
+	emailManagementAddGSuiteUsers,
+	emailManagementForwarding,
+	emailManagementManageTitanAccount,
+	emailManagementNewTitanAccount,
+	emailManagementTitanControlPanelRedirect,
+} from 'calypso/my-sites/email/paths';
+import EmailPlanMailboxesList from 'calypso/my-sites/email/email-management/home/email-plan-mailboxes-list';
+import {
+	getByPurchaseId,
+	hasLoadedSitePurchasesFromServer,
+	isFetchingSitePurchases,
+} from 'calypso/state/purchases/selectors';
+import QuerySitePurchases from 'calypso/components/data/query-site-purchases';
+import EmailPlanSubscription from 'calypso/my-sites/email/email-management/home/email-plan-subscription';
+import MaterialIcon from 'calypso/components/material-icon';
+import { resolveEmailPlanStatus } from 'calypso/my-sites/email/email-management/home/utils';
+import { getManagePurchaseUrlFor } from 'calypso/my-sites/purchases/paths';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+
+class EmailPlan extends React.Component {
+	static propTypes = {
+		domain: PropTypes.object.isRequired,
+		selectedSite: PropTypes.object.isRequired,
+
+		// Connected props
+		currentRoute: PropTypes.string,
+		hasEmailPlanSubscription: PropTypes.bool,
+		isLoadingPurchase: PropTypes.bool,
+		purchase: PropTypes.object,
+	};
+
+	state = {
+		isLoadingEmailAccounts: false,
+		errorLoadingEmailAccounts: false,
```

Comment:
Seems like we set this state property but we never use it. Maybe it can be removed?

Reply:
Thanks Marin! I will remove it in https://github.com/Automattic/wp-calypso/pull/52815.

Reply:
Thanks, @stephanethomas 🙌